### ISSUE TITLE
Window 2022 Black screen

### ISSUE THREAD

**Issue by 11v1** [actions/runner-images#7227]
### Description

We use GitHub actions to run automation tests that test our desktop application UI. FlaUI locator does not find the window, while it does in Windows 2019 image. I tried to record the video and everything I see is a black screen, no desktop, no windows, just a black screen. The video made in Windows 2019 has all of these.

### Platforms affected

- [ ] Azure DevOps
- [X] GitHub Actions - Standard Runners
- [X] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 18.04
- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] Windows Server 2019
- [X] Windows Server 2022

### Image version and build link

20230219.1

### Is it regression?

-

### Expected behavior

Explorer and windows persist on the video

### Actual behavior

No explorer and windows on the video

### Repro steps

-

**Comment by ddobranic** [comment]
@11v1 hello! could you please provide minimal repro steps?

**Comment by 11v1** [comment]
@ddobranic I'm sorry for the delay, had no chance to do the required. I have returned to this task again. My company granted me access to large runners with Windows 2019 meanwhile and I rechecked. So the results for now:
1. Windows 2019 default - works well
2. Windows 2019 large runner (any CPU number) - no desktop (black screen)
3. Windows 2022 default  - works well
4. Windows 2022 large runner (any CPU number) - no desktop (black screen)

I have created a public repo with workflow (just need to change the runner). It uploads the screen video to the artifacts: https://github.com/11v1/TestGithubActions
